packages/cdk8s-cli/lib/import/jsii.ts

```diff
@@ -41,16 +47,16 @@ export async function jsiiCompile(workdir: string, options: JsiiCompileOptions)
     name,
     version: '0.0.0',
     author: "dummy@dummy.com",
-    main: `${main}.js`,
-    types: `${main}.d.ts`,
+    main: moduleNamePrefix ? `${moduleNamePrefix}-${main}.js` : `${main}.js`,
+    types: moduleNamePrefix ? `${moduleNamePrefix}-${main}.d.ts` : `${main}.d.ts`,
```

Comment:
The typescript for this module is compiled at the location with `-` as separator. Below, in the `targets`, we use a `.` as separator.